### PR TITLE
Add To Do List with all functionalities

### PR DESCRIPTION
**Pull request by dicodiaz**
All functionalities are:

* Adding a new item.
* Removing a selected item.
* Marking a selected item as complete.
* Removing all items marked as complete at once.
* Reordering a selected item with drag-and-drop.
* Task.js includes the Task class with all its methods.
* images.js includes functions to append images.
* index.js includes windows.onload function.

### CODE REVIEW

File: src/Task.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
+import { appendCrossFadingImgs } from './images.js';
+import Menu from './menu.png';
+import Trash from './trash.png';
```
Comment (KatIsCoding) [inline src/Task.js]: [OPTIONAL] I see you decided to use your own images for the different Icons and this is great under normal conditions where you wanted to import your own artwork, however, the icons used can be found in some online icons providers, for example, Google Icons and Fontawesome which you should consider using more the next time you have to import an image, this can save you a few steps while developing and, at the same time demonstrate you know how to use such APIs!

File: src/Task.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    description,
+    completed = false,
+    index = Task.tasks.length + 1,
+    afterElementIndex = -1,
+  }) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+    this.afterElementIndex = afterElementIndex;
+  }
```
Comment (KatIsCoding) [inline src/Task.js]: - [x] I really like your OOP approach, however, it is kinda hard to follow up, especially because the `tasks` variable which is supposed to save all the different tasks is part of the `Task` class, this might be a small issue but it does great damage when trying to read and understand the code, maybe you should separate the methods regarding the `tasks` array from the actual `Task` class.
In a summarized conclusion, the `tasks` variable or the "Database" should be containing only the `Task` objects which are the ones that describe every task you will be storing, however, it doesn't make that much sense to save all the methods of `Task` class in every object, because anyways those methods only directly affect the whole `tasks` or database itself

Comment (KatIsCoding) [reply src/Task.js]: This refers to all your non-static methods (`populate`, `remove` and `add`)

File: src/images.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  topImg.src = topSource;
+  topImg.alt = `${container.id}-top`;
+  topImg.classList.add('cursor-pointer', 'cursor-move', 'img-fluid', 'opacity-0', 'position-absolute');
+  div.appendChild(botImg);
+  div.appendChild(topImg);
+  container.appendChild(div);
+  container.classList.add('position-relative');
+};
+
+export { appendImg, appendCrossFadingImgs };
```
Comment (KatIsCoding) [inline src/images.js:30]: - [ ] Great job on creating these methods for image adding, however, you have to remember this process can be simplified by using other methods such as using Icons providers

**Review by KatIsCoding — CHANGES_REQUESTED** [review]
## Hey! Your project is looking good! I really like the way you styled it and how you decided to use some OOP for some functionalities, however, I think there are some things I think should be changed, hopefully, you will be able to implement these changes and make your project look even better!

File: src/Task.js (hunk 2 above)
Comment (dicodiaz) [reply src/Task.js]: Thanks for your review! I implemented this.

**Review by KatIsCoding — APPROVED** [review]
## Good job applying the changes!! Now it will be way easier to maintain the code base of your project, congrats!! Now you can merge!
